index.html

```diff
@@ -2001,44 +2001,64 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 It is expected that securing mechanism specifications other than the ones
-referred to above will be created as necessary. A securing mechanism
-specification:
+referred to above will be created as necessary.
         </p>
 
-        <ul>
-          <li>
-MUST document normative algorithms that provide content integrity protection
-for <a>conforming documents</a>. The algorithms MAY be general in nature and
-MAY be used to secure data other than <a>conforming documents</a>.
-          </li>
-          <li>
-MUST provide a verification mechanism that returns only the information in the
-<a>conforming document</a> that has been secured. It MAY provide an
-additional mechanism to receive other information that might be helpful
-during validation or for debugging purposes. The concrete interface that is
-expected to be provided is detailed in the
-<a href="#verify-securing-mechanism">securing mechanism verification
-algorithm</a> section.
-          </li>
-          <li>
-MUST document post-processing considerations for data returned from verification
-processes. This includes, but is not limited to, what information can and cannot
-be used for decision making, for what period of time after verification, when
-additional information is required, and when care is necessary even when the
-returned data was properly secured.
-          </li>
-          <li>
-SHOULD provide integrity protection for any information referenced by a URL that
-is critical to validation. Mechanisms that can achieve this protection are
-discussed in Section <a href="#integrity-of-related-resources"></a> and Section
+        <p>
+Securing mechanism specifications MUST document normative algorithms that
+provide content integrity protection for <a>conforming documents</a>. The
+algorithms MAY be general in nature and MAY be used to secure data other than
+<a>conforming documents</a>.
+        </p>
+
+        <p>
+Securing mechanism specifications MUST provide a verification mechanism that
+returns only the information in the <a>conforming document</a> that has been
+secured. It MAY provide an additional mechanism to receive other information
+that might be helpful during validation or for debugging purposes. A securing
+mechanism's verification algorithm MUST provide an interface that
+receives an sequence of bytes ([=byte sequence=] |inputBytes|) and a media type
+([=string=] |inputMediaType|) as inputs and returns a verification result with
```

Comment:
I think it must be also possible for securing mechanisms to be valid if they accept a JSON document / parsed JSON document (Map/dictionary?) instead of bytes and an input media type. Many implementations will be written that way for embedded securing mechanisms (and already are), based on common software pipeline practices that will have JSON already being parsed into a document / object before it hits this API.

We should allow for either of these approaches to be used, especially if we are requiring these algorithms to be normative in the securing mechanism specifications.

Reply:
Fixed in 02c3744873614afe5ab0ae0cc14d5b0918c15916.